packages/cli/src/main.ts

```diff
@@ -60,6 +60,14 @@ export const createCLI = () => {
       '--name <value>',
       'name of plugin (Lowercase alphanumeric and dash)',
     )
+    .option(
```

Comment:
The approach looks good to me, Bibi. 🤔 

I wonder though, what if we have just the `--no-publish` option, and in case it's not passed (meaning it will be true), we always prompt the user to confirm the publishing unless the `--skipPrompts` is also present?

The combinations would be something like:

`yarn deploy` (which in case of an update, always asks if the user wants or not to publish)
`yarn deploy --skipPrompts` (publish without asking anything)
`yarn deploy --no-publish` (does not publish and does not show the prompt)
`yarn deploy --no-publish --skipPrompts` (does not publish and not show any prompt)

I didn't think deeply about it, so I may be missing some pieces 🤣 🤣

Reply:
This might also work. Let's see what @eunjae-lee and @Dawntraoz think as well :)

Reply:
This can be a slight breaking change. Let's imagine someone was using deploy command with all the options except for --skipPrompts, but since all the options were given, they didn't encounter any prompt, and they used it for their CI. Now we're changing the default behavior to prompt about publishing. Then their CI will break.

So I think we can keep publishing by default, and just provide `--no-publish`, then it would also be clear that publishing is done by default. What do you think? 🤔

Reply:
Ok so you would also suggest removing --publish? Works for me :) I will adjust it

Reply:
@demetriusfeijoo @eunjae-lee here are the updates 
a5a690e55c43a137992452afc4212f03b03a0401